db/migrate/20150309074119_add_deck_id_to_cards.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeckIdToCards < ActiveRecord::Migration
+  def change
+    add_reference   :cards, :deck, index: true
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.